lib/modules/manager/copier/schema.ts

```diff
@@ -1,10 +1,16 @@
 import { z } from 'zod';
+import { regEx } from '../../../util/regex';
 import { Yaml } from '../../../util/schema-utils';
 
+const GitSshUrl = z
+  .string()
+  .regex(regEx(/^(?<username>[^@]+)@[^:]*:(?<repository>.+)$/), {
```

Comment:
```suggestion
  .regex(regEx(/^[^@]+@[^:]*:.+$/), {
```
i don't see a benefit to capture groups, they're not used later